src/hope_payment_gateway/apps/gateway/models.py

```diff
@@ -30,6 +42,10 @@ class DeliveryMechanism(TimeStampedModel):
     )
     code = models.CharField(max_length=32, unique=True)
     name = models.CharField(max_length=128)
+    description = models.TextField(null=True, blank=True)
+    account_type = models.ForeignKey(
+        AccountType, on_delete=models.CASCADE, related_name="delivery_mechanisms", null=True, blank=True
```

Comment:
this should be a required field

Reply:
it will be... I'll clean the data and then remove